specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2025-04-01/ComputeRP.json

```diff
@@ -16089,6 +16127,30 @@
         }
       ]
     },
+    "ReservationType": {
+      "type": "string",
+      "description": "Indicates the type of capacity reservation. Allowed values are 'Block' for block capacity reservations and 'Targeted' for reservations that enable a VM to consume a specific capacity reservation when a capacity reservation group is provided. The reservation type is immutable and cannot be changed after it is assigned.",
+      "enum": [
+        "Targeted",
+        "Block"
+      ],
+      "x-ms-enum": {
+        "name": "ReservationType",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "Targeted",
+            "value": "Targeted",
+            "description": "Reservations that enable a VM to consume a specific capacity reservation when a capacity reservation group is provided."
+          },
+          {
+            "name": "Block",
+            "value": "Block",
+            "description": "Block capacity reservations."
```

Comment:
To consume scheduled allocated block capacity reservation when a capacity reservation group is provided.